node/pkg/dal/collector/utils.go

```diff
@@ -1,14 +1,15 @@
 package collector
 
 import (
+	"bytes"
 	"context"
 	"errors"
 	"time"
 
-	chainutils "bisonai.com/orakl/node/pkg/chain/utils"
 	"bisonai.com/orakl/node/pkg/chain/websocketchainreader"
-	"bisonai.com/orakl/node/pkg/reporter"
 
+	chainUtils "bisonai.com/orakl/node/pkg/chain/utils"
+	errorSentinel "bisonai.com/orakl/node/pkg/error"
```

Comment:
based on this [reference ](https://go.dev/blog/package-names), I'd recommend to use all lowercase for package naming, so `chainutils` instead of `chainUtils` and `errorsentinel` instead of `errorSentinel`